docs/vaccel_go.md

```diff
@@ -26,101 +26,56 @@ about Go installation can be found [here](https://go.dev/doc/install),
 but there are also instructions on how to install `Go` in the
 [vAccel-go bindings installation guide](https://github.com/nubificus/go-vaccel).
 
-### vAccel Go package
+## vAccel Go package
 The vaccel package in Golang provides access to vAccel operations, which
 can be used by the developers on their own `Go` programs. The vaccel package uses 
-the native `C` bindings in order to use the vAccel `C` API. The following diagram 
-demonstrates the functionality of the vaccel package:
+the native `C` bindings in order to use the vAccel `C` API.
 
-<figure>
-  <img src="img/vaccel_go.png" align=center />
-  <figcaption>Figure 1: High-level overview of the vAccel Go package</figcaption>
-</figure>
+## Installation Guide
 
-### Installation Guide
-#### vAccel Installation
+### vAccel Installation
 First of all, a vAccel installation is required before proceeding to the next sections. 
-
-#### Build from source
-In Ubuntu-based systems, you need to have the following packages to build vaccelrt:
-1. `cmake`
-2. build-essential
-
-You can install them using the following command:
-```
-sudo apt-get install -y cmake build-essential
-```
-Get the source code for **vaccelrt**:
+You can install `vaccel` using the following commands:
 ```
+sudo apt update
+sudo apt install --no-install-recommends python3-pip build-essential cargo pkg-config ninja-build libclang-dev
+sudo pip install meson
+
 git clone https://github.com/nubificus/vaccel --recursive
+cd vaccel
+meson setup -Dauto_features=enabled build
+meson compile -C build
+meson install -C build
 ```
-Prepare the build directory:
-```
-cd vaccelrt
-mkdir build
-cd build
-```
-#### Building the core runtime library
-```
-# This sets the installation path to /usr/local, and the current build
-# type to 'Release'. The other option is the 'Debug' build
-cmake ../ -DCMAKE_INSTALL_PREFIX=/usr/local -DCMAKE_BUILD_TYPE=Release -DBUILD_EXAMPLES=ON -DBUILD_PLUGIN_EXEC=ON -DBUILD_PLUGIN_NOOP=ON
-```
-```
-make
-sudo make install
-```
-#### vAccel-Go Bindings Installation
 
-##### Go Installation
-Of course, prior to installing the bindings, we have to make sure that Golang 1.20 or newer is installed in our system. We can check this using the following command:
+### vAccel-Go Bindings Installation
+
+Prior to installing the bindings, we have to make sure that Golang 1.20 or newer is installed in our system. We can check this using the following command:
```

Comment:
This should be "installed on" AFAIK

Reply:
Also Golang should be Go, I believe the project has transitioned from Golang to Go so we should use the latter for any references